### PR TITLE
fix: support <Show else> in builder json

### PR DESCRIPTION
**Pull request by manucorporat**

### CODE REVIEW

File: packages/core/src/generators/builder/generator.ts [hunk 1]
```diff
@@ -186,6 +186,57 @@ const componentMappers: {
...
+    const elseNode =
+      elseCase && filterEmptyTextNodes(elseCase)
+        ? el(
+            {
+              // TODO: the reverse mapping for this
+              component: {
+                name: 'Core:Fragment',
+              },
+              bindings: {
+                show: `!(${node.bindings.when?.code as string})`,
```
Comment (steve8708) [inline packages/core/src/generators/builder/generator.ts:217]: this alternatively can just use `hide`
```suggestion
                hide: node.bindings.when?.code as string,
```

File: packages/core/src/generators/solid/blocks.ts [hunk 2]
```diff
@@ -18,20 +19,26 @@ const transformAttributeName = (name: string) => {
...
-  if (json.properties._text) {
-    return json.properties._text;
-  }
-  if (json.bindings._text?.code) {
-    return `{${json.bindings._text.code}}`;
+export const blockToSolid = (
+  json: MitosisNode,
+  component: MitosisComponent,
+  options: ToSolidOptions,
+  insideJsx: boolean,
```
Comment (samijaber) [inline packages/core/src/generators/solid/blocks.ts:26]: can we keep the named arguments for this function? i find that much more readable than having to guess their order

**Review by samijaber — APPROVED** [review]
nice

File: packages/core/src/generators/solid/blocks.ts (hunk 2 above)
Comment (manucorporat) [reply packages/core/src/generators/solid/blocks.ts:26]: it's a ton of new new objects to GC, and solid it's the only one doing it

**Pull request merged**